coderd/database/queries/users.sql

```diff
@@ -223,6 +223,11 @@ WHERE
 			created_at >= @created_after
 		ELSE true
 	END
+	AND CASE
+		WHEN @github_com_user_id :: bigint != 0 THEN
+			github_com_user_id = @github_com_user_id
+		ELSE true
```

Comment:
I don't think this is worth blocking on, but the comment for this column mentions:

```
At time of implementation, this is used to check if the user has starred the Coder repository.
```

It may be worth updating this comment in a follow-up now that it's becoming more of a load-bearing number.